src/Chainweb/Mempool/InMem.hs

```diff
@@ -118,6 +124,33 @@ withInMemoryMempool cfg f = do
           f $! back
     bracket (makeInMemPool cfg) destroyInMemPool action
 
+withInMemoryMempool_ :: ToJSON t
+                     => FromJSON t
+                     => Logger logger
+                     => logger
+                     -> InMemConfig t
+                     -> (MempoolBackend t -> IO a)
+                     -> IO a
+withInMemoryMempool_ l cfg f = do
+    let action inMem = do
+          r <- race (monitor inMem) $ do
+            back <- toMempoolBackend inMem
+            f $! back
+          case r of
+            Left () -> throw $ InternalInvariantViolation "mempool monitor exited unexpectedly"
+            Right result -> return result
+    bracket (makeInMemPool cfg) destroyInMemPool action
+  where
+    monitor m = do
+        let lf = logFunction l
+        logFunctionText l Info $ "Initialized Mempool Monitor"
+        runForeverThrottled lf "Chainweb.Mempool.InMem.withInMemoryMempool_.monitor" 10 (10 * mega) $ do
+            stats <- getMempoolStats m
+            logFunctionText l Debug $ "got stats"
+            logFunctionJson l Info stats
+            logFunctionText l Debug $ "logged stats"
+            threadDelay 60000000 {- 1 minute -}
```

Comment:
We should randomize this interval slightly -- otherwise all of the threads are going to all wake up at the same time causing lumpy load spikes. (This is a problem throughout the codebase, but since we fork a monitor per chain it will be bad here)